src/Libraries/Microsoft.Extensions.Caching.Hybrid/Microsoft.Extensions.Caching.Hybrid.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Multi-level caching implementation building on and extending IDistributedCache</Description>
+    <TargetFrameworks>$(NetCoreTargetFrameworks)$(ConditionalNet462);netstandard2.0;netstandard2.1</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>cache;distributedcache;hybrid</PackageTags>
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
+    <InjectIsExternalInitOnLegacy>true</InjectIsExternalInitOnLegacy>
+    <InjectCallerAttributesOnLegacy>true</InjectCallerAttributesOnLegacy>
+    <InjectTrimAttributesOnLegacy>true</InjectTrimAttributesOnLegacy>
+    <InjectExperimentalAttributeOnLegacy>true</InjectExperimentalAttributeOnLegacy>
+    <InjectSharedDiagnosticIds>true</InjectSharedDiagnosticIds>
+    <IsPackable>true</IsPackable>
+
+    <!-- ignore "use [Experimental] for new API" -->
+    <NoWarn>$(NoWarn),LA0003</NoWarn>
```

Comment:
Instead of this, can we just add a json file on this same directory with the existing API marked as stable? That would silence the analyzer. Those files can be easily generated using this tool in the repo: https://github.com/dotnet/extensions/tree/main/eng/Tools/ApiChief which you can either run manually, or alternatively you can run the script that invokes it: https://github.com/dotnet/extensions/blob/main/scripts/MakeApiBaselines.ps1